api/tsconfig.json

```diff
@@ -1,4 +1,7 @@
 {
-  "extends": ["../tsconfig.json"],
-  "include": ["src/**/*"]
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "dist"
+  },
+  "include": ["src"]
```

Comment:
Still keeping this file around as vscode will still use it when needed. But we use `.swcrc` to build